src/dmd/root/file.d

```diff
@@ -20,14 +20,15 @@ import core.sys.posix.unistd;
 import core.sys.windows.winbase;
 import core.sys.windows.winnt;
 import dmd.root.filename;
+import dmd.root.outbuffer;
 import dmd.root.rmem;
 import dmd.utils;
 
 /***********************************************************
  */
 struct File
 {
-    int _ref; // != 0 if this is a reference to someone else's buffer
+    bool _ref; // true if this is a reference to someone else's buffer
```

Comment:
Perhaps it's time to make this more descriptive, since you're changing it everywhere anyway. Perhaps to `ownedByThis` ? (Note that reverses the sense of it, too.)